### ISSUE TITLE
Predicted Label to Predicted token mapping in LayoutLMV3. Please look at the description.

### ISSUE THREAD

**Issue by slk-genai** [huggingface/transformers#27424]
### Feature request

I am trying the below code:

processor = LayoutLMv3Processor.from_pretrained("/content/layoutlmv3-large-finetuned-funsd")
encoding = processor(image, return_offsets_mapping=True, return_tensors="pt")
offset_mapping = encoding.pop('offset_mapping')
print(encoding.keys())
device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
for k,v in encoding.items():
  encoding[k] = v.to(device)
model = LayoutLMv3ForTokenClassification.from_pretrained("/content/layoutlmv3-large-finetuned-funsd")
model.to(device)
outputs = model(**encoding)
print(outputs.logits.shape)

I can see that there are parts of image which are getting labelled but during OCR identification those texts are not getting identified. 
So, how can I get the label and their respective token?

### Motivation

I am new to LayoutLMv3 and I have task in hand which was getting easily done but I can't find a way to map the token prediction and it's respective label prediction mapping. I am not using the above checkpoints but the problem is similar to the mentioned checkpoint. 

### Your contribution

Let me know if you need any help.

**Comment by ArthurZucker** [comment]
Hey 🤗  thanks for opening an issue! We try to keep the github issues for bugs/feature requests.
Could you ask your question on the [forum](https://discuss.huggingface.co/) instead? I'm sure the community will be of help!

Thanks!
